docs/api-reference/next.config.js/build-target.md

```diff
@@ -16,7 +16,7 @@ Your application will be built and deployed as a monolith. This is the default t
 
 ## `serverless` target
 
-> Deployments to [ZEIT Now](https://zeit.co) will automatically enable this target. You do not need to opt-into it yourself, but you can.
+> Deployments to [ZEIT Now](https://zeit.co) will automatically enable this target. You should not opt-into it yourself.
```

Comment:
Should we mention if they want to debug something locally they should use the actual target we use, `experimental-serverless-trace`?

Reply:
I was hesitant to add something with `experimental` in the name. I'm pretty confident in its ability such that we don't need to explain how to troubleshoot in the docs.

Reply:
Also, `experimental-serverless-trace` should realistically never fail when run locally (in cases that `server` would succeed). The actual tracing (which you'd need to troubleshoot) is an implementation detail of our builder.